org.aardappel.TreeSheets.yml

```diff
@@ -0,0 +1,26 @@
+app-id: org.aardappel.TreeSheets
```

Comment:
`aardappel.org` doesn't seem to response. is this domain under the control of the upstream developers?

Reply:
Ok, changed that to the correct rDNS scheme.